### DISCUSSION TITLE
What type of error is this

### DISCUSSION THREAD

**Discussion by ChillVibesMushroom** [aircrack-ng/aircrack-ng#2537] [Q&A]
Network controller: Realtek Semiconductor Co., Ltd. RTL8821CE 802.11ac PCIe Wireless Network Adapter
Linux kali 5.18.0-kali5-amd64 #1 SMP PREEMPT_DYNAMIC Debian 5.18.5-1kali6 (2022-07-07) x86_64 GNU/Linux
Aircrack-ng 1.7  - (C) 2006-2022 Thomas d'Otreppe
global
country US: DFS-FCC
        (902 - 904 @ 2), (N/A, 30), (N/A)
        (904 - 920 @ 16), (N/A, 30), (N/A)
        (920 - 928 @ 8), (N/A, 30), (N/A)
        (2400 - 2472 @ 40), (N/A, 30), (N/A)
        (5150 - 5250 @ 80), (N/A, 23), (N/A), AUTO-BW
        (5250 - 5350 @ 80), (N/A, 24), (0 ms), DFS, AUTO-BW
        (5470 - 5730 @ 160), (N/A, 24), (0 ms), DFS
        (5730 - 5850 @ 80), (N/A, 30), (N/A), AUTO-BW
        (5850 - 5895 @ 40), (N/A, 27), (N/A), NO-OUTDOOR, AUTO-BW, PASSIVE-SCAN
        (5925 - 7125 @ 320), (N/A, 12), (N/A), NO-OUTDOOR, PASSIVE-SCAN
        (57240 - 71000 @ 2160), (N/A, 40), (N/A)

This is the issue I am having sometimes not all the time and it's mind boggling I even went all the way to re installing the drivers for kali 
which never had to be done in the past my network bro is too good...

should I use the --cswitch method and choose a different channel hopping method idk whats going on 

https://user-images.githubusercontent.com/99580833/229301606-91b38b57-4b92-4a61-95f3-58d091b54533.mp4

**Comment by ChillVibesMushroom** [comment]
This is also happening sometimes with my 2.4 Ghz alfa atheros driver only sometimes and it's mind boggling

**Comment by Mister-X-** [comment]
I'm not watching the whole video. You are going to have to describe your issue with words. And put outputs and commands between backticks, so this is readable.
